src/Sindarin-Tests/SindarinDebuggerTest.class.st

```diff
@@ -362,16 +362,10 @@ SindarinDebuggerTest >> testIsExecutionFinished [
 	| scdbg |
 	scdbg := SindarinDebugger debug: [ self helperMethod16 ].
 	self deny: scdbg isExecutionFinished.
-	scdbg
-		stepOver;
-		step;
-		step;
-		stepOver;
-		step;
-		stepOver;
-		step.
-	"Reached `self isActiveProcess` in Process>>#terminate"
-	self assert: scdbg isExecutionFinished
+	
+	[ scdbg isExecutionFinished ] whileFalse: [ scdbg stepOver ].
+
+	self assert: scdbg currentProcess isTerminated
```

Comment:
This change makes test independent from the actual process stack during the self termination.

@StevenCostiou please sync these changes back to actual repository if it exists